src/lib/db.js

```diff
@@ -342,9 +342,17 @@ export default class db {
     return new Promise((resolve) => {
       database.select('slug')
       .from('teams')
-      .orderBy('name', 'asc')
+      .innerJoin('teams_authors', 'teams.id', '=', 'teams_authors.team_id')
+      .orderBy('teams_authors.team_order', 'asc')
       .then((rows) => {
-        const result = _.map(rows, (row) => {return row.slug});
+        // The SQL is returning a copy of the team for every author. This
+        // solution manually filters the results and works but it's hacky.
+        // We don't use ES6 sets to preserve the order.
```

Comment:
Should we bother though? Running time will always be linear because we have to check each result at least once, and we won't every get to a point of having thousands of staff members where you can start to feel the difference.